content/censorship/using-snowflake-with-Tor-browser-alpha/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: How can you use Snowflake with Tor browser alpha?
+---
+seo_slug: How-can-you-use-Snowflake-with-Tor-browser-alpha
```

Comment:
Please rename seo_slug, directory name and title:
A) We should have a shorter URL
B) In a few months when it's stable, we don't need to rewrite the URL, Title and seo_slug.
For example: How can I use Snowflake with Tor Browser?
URL/directory name: snowflake-tor-browser